ooniprobe/PrefixHeader.pch

```diff
@@ -14,7 +14,7 @@
 #define NOTIFICATION_SERVER @"https://registry.proteus.ooni.io"
 #endif
 
-#define release_name @""
+#define release_name @"-rc.2"
```

Comment:
@lorenzoPrimi just checking in to make sure I did not mistakes in fixing conflcts: okay to have `-rc.2` here?